README.md

```diff
@@ -25,15 +25,9 @@ animal.name === 'jon'
 
 ## API
 
-### merge(destination, source)
+### merge(Object destination, Object source [, bool overwrite])
```

Comment:
The style here is that this is actual JavaScript. Please update the header to at least be valid JavaScript still 👍

Reply:
ok.  Do you prefer to maintain two distinct definitions with/without the third argument?

Reply:
You could put a variable instead as the third argument. But as long as it's obvious that the third argument can just be omitted in the function call.